### PR TITLE
new plugins

### PR DESCRIPTION
**Pull request by muhammedfurkan**

### CODE REVIEW

File: pyrobot/__init__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 if bool(os.environ.get("ENV", False)):
     from pyrobot.sample_config import Config
 else:
-    from pyrobot.config import Development as Config
+    from pyrobot.sample_config import Development as Config
```
Comment (SpEcHiDe) [inline pyrobot/__init__.py]: what?

Comment (muhammedfurkan) [reply pyrobot/__init__.py]: no need rename sample_config to config..

Comment (SpEcHiDe) [reply pyrobot/__init__.py]: ?

File: pyrobot/plugins/__init__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        basename(f)[:-3]
+        for f in mod_paths
+        if isfile(f) and f.endswith(".py") and not f.endswith("__init__.py")
+    ]
+    return all_modules
+
+
+ALL_MODULES = sorted(__list_all_modules())
+logger.info("Modules to load:\n{}\n".format(", ".join(ALL_MODULES)))
+__all__ = ALL_MODULES + ["ALL_MODULES"]
```
Comment (SpEcHiDe) [inline pyrobot/plugins/__init__.py]: we don't need the @banHammerMarie_bot way of imports. We are using https://docs.pyrogram.org/topics/smart-plugins which is documented officially.

**Pull request closed without merging**